### PR TITLE
improve text syntax

### PR DESCRIPTION
**Pull request by lebogangolifant**
**Summary**

improve text syntax

### CODE REVIEW

**Comment by mrkev** [comment]
Hey @lebogangolifant— when I import the PR the summary is what shows up as the commit message. Could you edit it and make it meaningful to the change? Doesn't have to be long, just not the template.

**Comment by lebogangolifant** [comment]
@mrkev thanks for reviewing my PR. I have made changes according to your suggestion  I hope this will help.

File: docs/APIReference-APIMigration.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-id: v0-10-api-migration
+id: v0.10 API Migration
```
Comment (mrkev) [inline docs/APIReference-APIMigration.md:2]: Hmm, if we change the ID here won't we need to change all the places we reference it? Does this change the URL too? Maybe it's a good idea to leave it as is.

**Comment by lebogangolifant** [comment]
@mrkev  I referred to Docusaurus API specification docs on how to write down markdown header features, and you are correct. So I changed back the id to its original state. Thanks for highlighting this, learned something new.
API migration docs

**Pull request closed without merging**